client-app/src/pages/account/store/modules/invoices-list/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Module } from "vuex";
+import { defaultPageSize, invoicesGridFields, invoicesStatuses } from '@common/constants';
+import { localizeGridColumns } from '@common/services/localization.helper.service';
+import { RootState } from "../../types";
+import { actions } from "./actions";
+import { getters } from "./getters";
+import { mutations } from "./mutations";
+import { InvoicesListState } from "./types";
+
+
+// initial state
+export const initialState: InvoicesListState = {
+  errors: {},
+  invoicesList: {
+    listConfig: {
+      columns: invoicesGridFields,
+      pageNumber: 1,
+      pageSize: defaultPageSize,
```

Comment:
Why are you using the defaultPageSize here but don't use the same for pageNumber? :)
It's a pretty infrequent scenario when we will have to change the defaultPageNumber, but what if we will?
Besides, it's better to don't use the magic numbers at all